packages/fxa-payments-server/src/routes/Product/SubscriptionRedirect/index.tsx

```diff
@@ -11,16 +12,16 @@ export type SubscriptionRedirectProps = {
   plan: Plan;
 };
 
-export const SubscriptionRedirect = ({
-  plan: { product_id, product_name },
-}: SubscriptionRedirectProps) => {
+export const SubscriptionRedirect = ({ plan }: SubscriptionRedirectProps) => {
+  const { product_id, product_name } = plan;
```

Comment:
We could remap all these variable locally, but will not be changing them in the upstream API from which these values come. I didn't bother here because it didn't really seem to make a practical difference.

Reply:
Ah, yeah, agreed. Might just be something we need to deal with if Stripe consistently uses snakecase.